src/project-css/public/js/equalheight.js

```diff
@@ -0,0 +1,13 @@
+function equalHeight(group) {
+   tallest = 0;
+   group.each(function() {
+      thisHeight = $(this).height();
+      if(thisHeight > tallest) {
+         tallest = thisHeight;
+      }
+   });
+   group.height(tallest);
+}
+$(document).ready(function() {
```

Comment:
What happns on screen resize?  Probably nothing looking at this JS.  But if you drag your screen you should bind it to a resize event so that the method runs again.